src/components/text-editor/prosemirror-adapter/prosemirror-adapter.tsx

```diff
@@ -224,11 +226,21 @@ export class ProsemirrorAdapter {
         const tr = this.view.state.tr;
         tr.replaceWith(0, tr.doc.content.size, prosemirrorDoc.content);
         this.view.dispatch(tr);
+        this.suppressChangeEvent = false;
```

Comment:
Should this really be set here? It seem like a better fit to clear this boolean on row 237. I can't see any relevant events here that would make `handleTransaction` to be called before flipping this boolean

Reply:
But since this seems to have solved it for you guys I'm probably missing somehting 😅

Reply:
I've looked at it, and I _think_ I disagree. It feels to me like `handleTransaction` shouldn't just assume that because we suppressed one change event, we should let the next one trough. I think it's better that the function that sets the value also clears it.

With the current state of the code, it shouldn't matter in practice, but I think it feels cleaner for potential future changes to not have `handleTransaction` making assumptions like that.

Reply:
I agree with this. For some reason I assumed that the dispatch somehow was hiding some async code but it must not then.